web/yo/app/scripts/directives/driverealtimestring.js

```diff
@@ -119,13 +119,23 @@ angular.module('oncokbApp')
                         scope.content.propagation = scope.es.get('propagation');
                     }
                     scope.changePropagation(true);
-                    scope.$watch('content.propagation', function(n) {
+                    scope.$watch('content.propagation', function(n, o) {
                         if (!_.isUndefined(scope.es)) {
                             if (_.isUndefined(scope.es.get('propagation')) ||
                                 scope.es.get('propagation') !== n) {
                                 scope.es.set('propagation', n);
                             }
                         }
+                        scope.rs.set('lastReviewedPropagation', o);
```

Comment:
If o is null, that means this is the first time to set up the propagation info for the treatment, and usually, that means it's setting up the default value. In this situation, you don't have to trigger the review model.